HISTORY.md

```diff
@@ -4,6 +4,14 @@ In the beginning, programmers created the databases. Now the databases were form
 
 And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.
 
+## Snowfakery 0.8.1
+
+Snowfakery includes support for Python RecordTypes.
+
+Snowfakery can output SQL/JSON NULL using Yaml blank fields or the YAML literal 'null'
+
+Various performance and reliability improvements.
```

Comment:
Is there a good reason to be vague about it? I always hate it when release notes don't actually say what changed.